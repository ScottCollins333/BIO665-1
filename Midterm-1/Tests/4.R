source("4.R")

# The first few lines of code generate random numbers

set.seed(0)

dbh1_a = rnorm(20, mean=20, sd=5)
dbh1_b = rnorm(20, mean=20, sd=5)

dbh2_a = c(dbh1_a, dbh1_a + runif(20) * 2)
dbh2_b = c(dbh1_b, dbh1_b + runif(20) * 2)

print(getBasalArea(dbh1_a[1]))
print(getBasalArea(dbh1_b[1]))

print(getBasalAreaVector(dbh1_a))
print(getBasalAreaVector(dbh1_b))

print(getTwoYearAverage(dbh2_a))
print(getTwoYearAverage(dbh2_b))

print(getLogicalGrowth(dbh2_a))
print(getLogicalGrowth(dbh2_b))