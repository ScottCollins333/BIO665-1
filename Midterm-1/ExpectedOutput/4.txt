[1] 3.776823
[1] 1.943881
 [1] 3.7768227 1.8403081 3.8733711 3.7904339 2.6574082 0.8251925 1.2863214
 [8] 1.8720151 2.1753751 5.5931783 3.0941173 1.3971281 1.1093559 1.8773328
[15] 1.8674762 1.7558632 2.4654690 1.3171992 2.6828010 1.0405427
 [1] 1.943881 2.612757 2.329533 3.147079 2.119812 2.765595 3.526799 1.493046
 [9] 1.005392 2.232930 1.932121 1.629650 1.734595 1.530714 3.046441 3.619130
[17] 3.398165 1.738290 3.741526 1.887582
 [1] 4.062634 1.929463 4.086034 3.907204 2.736902 0.933027 1.320725 2.021243
 [9] 2.202047 5.679590 3.131574 1.439591 1.118562 2.011818 2.052724 1.914932
[17] 2.657318 1.396638 2.783847 1.169888
 [1] 2.072450 2.773748 2.410518 3.218692 2.344040 2.925580 3.586436 1.516577
 [9] 1.078695 2.446201 2.058964 1.824107 1.882794 1.597291 3.159708 3.661016
[17] 3.401728 1.883225 3.771123 1.980319
 [1]  TRUE FALSE  TRUE FALSE FALSE  TRUE FALSE  TRUE FALSE FALSE FALSE FALSE
[13] FALSE  TRUE  TRUE  TRUE  TRUE FALSE FALSE  TRUE
 [1]  TRUE  TRUE FALSE FALSE  TRUE  TRUE FALSE FALSE FALSE  TRUE  TRUE  TRUE
[13]  TRUE FALSE FALSE FALSE FALSE  TRUE FALSE FALSE
